stock_picking_report_delivery_cost/tests/test_stock_picking_report_delivery_cost.py

```diff
@@ -52,7 +52,7 @@ def setUpClass(cls):
     def test_carrier_price_for_report_before(self):
         delivery_wizard = Form(
             self.env["choose.delivery.carrier"].with_context(
-                {
+                **{
```

Comment:
```
self.env["choose.delivery.carrier"].with_context(
    default_order_id=self.order.id,
    default_carrier_id=self.carrier.id,
)
```

Reply:
Much better, clearer and cleaner, thanks for the suggestion.